Sources/Account/Account+SCIOperations.swift

```diff
@@ -0,0 +1,169 @@
+            metaFetcher: BlockchainMetaFetcher,
+            txOutSelectionStrategy: TxOutSelectionStrategy,
+            mixinSelectionStrategy: MixinSelectionStrategy,
+            rngSeed: RngSeed,
+            targetQueue: DispatchQueue?
+        ) {
+            self.serialQueue = DispatchQueue(
+                label: "com.mobilecoin.\(Account.self).\(Self.self))",
+                target: targetQueue)
+            self.account = account
+            self.metaFetcher = metaFetcher
+            self.txOutSelector = TxOutSelector(txOutSelectionStrategy: txOutSelectionStrategy)
+            self.signedContingentInputCreator = SignedContingentInputCreator(
+                accountKey: account.accessWithoutLocking.accountKey,
+                fogMerkleProofService: fogMerkleProofService,
+                fogResolverManager: fogResolverManager,
+                mixinSelectionStrategy: mixinSelectionStrategy,
+                rngSeed: rngSeed,
+                targetQueue: targetQueue)
+        }
+
+        private func verifyBlockVersion(
+            _ blockVersion: BlockVersion,
+            _ completion: @escaping (
+                Result<SignedContingentInput, SignedContingentInputCreationError>) -> Void
+        ) -> Bool {
+            // verify block version >= 3
+            guard blockVersion >= 3 else {
+                serialQueue.async {
+                    completion(.failure(.requiresBlockVersion3(
+                        "Block version must be > 3 for SCI support")))
+                }
+                return false
+            }
+            return true
+        }
+
+        private func verifyAmountIsNonZero(
+            _ amount: Amount,
+            _ actionDescription: String,
+            _ completion: @escaping (
+                Result<SignedContingentInput, SignedContingentInputCreationError>) -> Void
+        ) -> Bool {
+            guard amount.value > 0 else {
+                let errorMessage = "createSignedContingentInput failure: " +
+                    "Cannot \(actionDescription) 0 \(amount.tokenId)"
+                logger.error(errorMessage, logFunction: false)
+                serialQueue.async {
+                    completion(.failure(.invalidInput(errorMessage)))
+                }
+                return false
+            }
+            return true
+        }
+
+        private func logTxOuts(_ txOuts: [KnownTxOut], _ message: String) {
+            logger.info(
+                "\(message): " +
+                    """
+                        0x\(redacting: txOuts.map {
+                            $0.publicKey.hexEncodedString()
+                        })
+                    """,
+                logFunction: false)
+        }
+
+        private func logUnspentTxOuts(
+            _ recipient: PublicAddress,
+            _ amountToSend: Amount,
+            _ amountToReceive: Amount,
+            _ txOuts: [KnownTxOut]
+        ) {
+            logger.info(
```

Comment:
since these functions don't use any member variables can we move them to the logger file `MobileCoinLogging` ?

Reply:
Seems like MobileCoinLogging is much more generic / reusable / general purpose...  

This was code that was inline originally (starting code taken from Account+TransactionOperations.swift, where it's inlined there)... not sure why I pulled it out, maybe was just trying to simplify the body.

I re-inlined it for now, but I don't think that MobileCoinLogging is the right place... maybe someplace else if you don't like the inlining - like an extension at bottom or in another related filename for A+SCOOps